### DISCUSSION TITLE
[SOLVED] Youtube subscriptions CSV to OPML converter

### DISCUSSION THREAD

**Discussion by formlessdao** [martinrotter/rssguard#668] [Q&A]
Hi, google's service named Google Takeout exports all my youtube subscriptions as a CSV file. What can I do to convert it to a valid OPML file?

What file types does RSS Guard support besides OPML to import feeds or just like in my case import that CSV file generated by google itself.

**Comment by formlessdao** [comment]
Earth to Martin Rotter... do you read?

**Reply by martinrotter** [reply]
Covid

**Reply by formlessdao** [reply]
> Covid

Sorry I can't read minds, one word replies can not be considered replies.

**Reply by martinrotter** [reply]
I have covid.

Anyway, CSV is not support, but plain text 1 URL per line is. So file like
```
http://www.caaaamcqwlůcmqwmůmům...../feed
http://www.caaaamcqwlůcmqwmůmům...../feed
http://www.caaaamcqwlůcmqwmůmům...../feed
```

woult get parsed OK.

**Reply by martinrotter** [reply]
![image](https://user-images.githubusercontent.com/1255302/164203787-2cdb6f1f-d7ca-4680-b141-ebf92ac52f87.png)
